code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -652,7 +652,7 @@
 		playsound(loc, 'sound/magic/charge.ogg', 50, 1)
 		if(current_charges == max_charges)
 			playsound(loc, 'sound/machines/ding.ogg', 50, 1)
-			processing_objects.Remove(src)
+			STOP_PROCESSING(SSobj, src)

```

Comment:
Note to self, see if you can actually add the proper shielded overlays for these now that we have SSoverlays.